geolib/models/dstability/internal.py

```diff
@@ -77,16 +77,16 @@ class PersistablePoint(DStabilityBaseModelStructure):
 
 class PersistableHeadLine(DStabilityBaseModelStructure):
```

Comment:
There is quite some inconsistency in this file. Everything seems `Optional`, which I think is not always the case. And then these sometimes still get a default value assigned, which doesn't necessarily have to make sense (height = 0, width = 0). I'd like to see a future improvement story that makes this model consistent.

Reply:
https://issuetracker.deltares.nl/browse/DSTAB-1580